andela/zinnia-ah-backend
# #164797137 Chore Configured eslint

#### What does this PR do?
Setup and configured eslint.
using **Airbnb-base** as the **eslint** standard

#### Description of Task to be completed?

#### How should this be manually tested?
* clone `https://github.com/andela/zinnia-ah-backend.git`
* checkout to this branch = `ch-configure-eslint-164797137`
* pull from the remote branch to update the local branch
* run `npm install`
* `console.log()` a message in any **.js** file
* **eslint** should flag an error

#### Any background context you want to provide? 
none

#### What are the relevant pivotal tracker stories? 
#164797137

#### Screenshots (if appropriate)
none

#### Questions:
none
